CMakeLists.txt

```diff
@@ -128,6 +139,14 @@ if("${CMAKE_CXX_COMPILER_LAUNCHER}" STREQUAL "ccache" AND
     add_compile_options(-fdiagnostics-color=always)
 endif()
 
+if(NOT WIN32)
+  if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
Can you change this block's location to https://github.com/eProsima/Fast-RTPS/blob/master/CMakeLists.txt#L119?

Reply:
Done in https://github.com/eProsima/Fast-RTPS/pull/229/commits/4db96d431a914bf8712df542a41ced3a9324a781